scripts/convert.py

```diff
@@ -54,15 +54,17 @@ def make_groups(words: Words) -> Groups:
     return groups
 
 
+# For each word, we have a variant field if the word is "pure verb form" (it does have other section than "Forme de verbe")
 def make_variants(words: Words) -> Variants:
     """Group word by variant for later HTML creation, see save()."""
     variants: Variants = defaultdict(list)
     for word, details in words.items():
         details = Word(*details)
         # Variant must be normalized by trimming whitespace and lowercasing it.
-        variant = details.variant.lower().strip()
-        if variant:
-            variants[variant].append(word)
+        for variant in details.variants:
+            variant = variant.lower().strip()
```

Comment:
Maybe this transformation could de moved to `get.py`, when a variant is actually added?
Because the same sanitization has to be done L191, and we should prevent such duplicate code.

Reply:
Or we do it only when we write the variants to the HTML files ?

Reply:
I would vote for doing that at the root, but it is your call :)

Reply:
It's safer at the end. We do comparison during the process and it's only at the end that we really need it lowercase.